src/components/Proposal/Create/PluginForms/CreateTokenTradeProposal.tsx

```diff
@@ -0,0 +1,325 @@
+  handleClose: () => any;
+  pluginState: IPluginState;
+}
+
+const mapStateToProps = (state: IRootState, ownProps: IExternalProps) => {
+  return ownProps;
+};
+
+interface IDispatchProps {
+  createProposal: typeof arcActions.createProposal;
+  showNotification: typeof showNotification;
+}
+
+const mapDispatchToProps = {
+  createProposal: arcActions.createProposal,
+  showNotification,
+};
+
+type IProps = IExternalProps & IDispatchProps;
+
+interface IFormValues {
+  description: string;
+  title: string;
+  url: string;
+  tags: Array<string>;
+  sendTokenAddress: string,
+  sendTokenAmount: number,
+  receiveTokenAddress: string,
+  receiveTokenAmount: number,
+}
+
+interface IState {
+  tags: Array<string>;
+}
+
+class CreateKnownPluginProposal extends React.Component<IProps, IState> {
+
+  initialFormValues: IFormValues;
+
+  constructor(props: IProps) {
+    super(props);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+
+    this.initialFormValues = importUrlValues<IFormValues>({
+      description: "",
+      title: "",
+      url: "",
+      tags: [],
+      sendTokenAddress: "",
+      sendTokenAmount: 0,
+      receiveTokenAddress: "",
+      receiveTokenAmount: 0,
+    });
+
+    this.state = {
+      tags: this.initialFormValues.tags,
+    };
+  }
+
+  public async handleSubmit(values: IFormValues, { setSubmitting }: any ): Promise<void> {
+    if (!await enableWalletProvider({ showNotification: this.props.showNotification })) { return; }
+
+    const proposalOptions: IProposalCreateOptionsTokenTrade = {
+      dao: this.props.daoAvatarAddress,
+      descriptionHash: values.description,
+      title: values.title,
+      tags: this.state.tags,
+      plugin: this.props.pluginState.address,
+      url: values.url,
+      sendTokenAddress: values.sendTokenAddress,
+      sendTokenAmount: values.sendTokenAmount,
+      receiveTokenAddress: values.receiveTokenAddress,
+      receiveTokenAmount: values.receiveTokenAmount,
+    };
+
+    setSubmitting(false);
+
+    await this.props.createProposal(proposalOptions);
+
+    Analytics.track("Submit Proposal", {
+      "DAO Address": this.props.daoAvatarAddress,
+      "Proposal Title": values.title,
+      "Plugin Address": this.props.pluginState.address,
+      "Plugin Name": this.props.pluginState.name,
+    });
+
+    this.props.handleClose();
+  }
+
+  private onTagsChange = (tags: any[]): void => {
+    this.setState({tags});
+  }
+
+  public exportFormValues(values: IFormValues) {
+    values = {
+      ...values,
+      ...this.state,
+    };
+    exportUrl(values);
+    this.props.showNotification(NotificationStatus.Success, "Exportable url is now in clipboard :)");
+  }
+
+  public render(): RenderOutput {
+    const { handleClose } = this.props;
+
+    const fnDescription = () => (<span>Short description of the proposal.<ul><li>What are you proposing to do?</li><li>Why is it important?</li><li>How much will it cost the DAO?</li></ul></span>);
```

Comment:
Same here, please use `i18next`.
see line 578 in `CreatePluginManagerProposal.tsx`